pkgs/shells/fish/default.nix

```diff
@@ -148,6 +148,20 @@ let
       hash = "sha256-YUyfVkPNB5nfOROV+mu8NklCe7g5cizjsRTTu8GjslA=";
     };
 
+    patches = [
+      # We don’t want to run `/usr/libexec/path_helper` on nix-darwin,
+      # as it pulls in paths not tracked in the system configuration
+      # and messes up the order of `$PATH`. Upstream are unfortunately
+      # unwilling to accept a change for this and have recommended that
+      # it should be a distro‐specific patch intesad.
```

Comment:
```suggestion
      # it should be a distro‐specific patch instead.
```

Reply:
Intesad, is that latin?

Reply:
To have to apply such a patch makes me into sad. :(

Fixed.